src/Authentication/Authentication/Properties/Resources.resx

```diff
@@ -199,10 +199,19 @@
   <data name="NotConnectedToGraphException" xml:space="preserve">
     <value>The cmdlet cannot run because Authentication is set to {0} and Connect-Graph was not called. Invoke 'Connect-Graph' or specify Authentication to be 'UserProvidedToken' and Provide a Token then retry</value>
   </data>
+  <data name="MissingAuthenticationContext" xml:space="preserve">
+    <value>The cmdlet cannot run because {0} is missing and 'Authentication' is set to 'Default'. Please call 'Connect-Graph' then try again.</value>
```

Comment:
```suggestion
    <value>The cmdlet cannot run because {0} is missing and 'Authentication' is set to 'Default'. Please call 'Connect-MgGraph' then try again.</value>
```
nit: Let's avoid using aliases where possible since they could change.